I/IGLWrap/iglwrap/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.16)
+project(iglwrap)
+
+list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+
+# Boost
+set(Boost_USE_STATIC_LIBS OFF)
+set(CMAKE_BUILD_TYPE Release)
+find_package(Boost 1.71 COMPONENTS thread REQUIRED)
+#  include(FindBoost)
+
+# Libigl
+option(LIBIGL_WITH_OPENGL            "Use OpenGL"         OFF)
+option(LIBIGL_WITH_OPENGL_GLFW       "Use GLFW"           OFF)
+option(LIBIGL_WITH_CGAL "Use CGAL" ON)
```

Comment:
Don't you need to add `CGAL_jll` as dependency?

Reply:
I don't think so: the build script shows that `cgal` is somehow downloaded anyway (I suspect `cmake` magic from the `libigl` install scripts).

Reply:
> the build script shows that cgal is somehow downloaded anyway

That's precisely what we _don't_ want :slightly_smiling_face: